### DISCUSSION TITLE
Who watched this PR?

### DISCUSSION THREAD

**Discussion by sospedra** [community/community#35091] [Pull Requests]
I think it'd be very useful to know who watched a PR. Similar like Notion does with this: 
<img width="262" alt="image" src="https://user-images.githubusercontent.com/3116899/193849424-c220891c-9749-4268-9f73-21413576000f.png">

Maybe not in public PRs but for company private PRs.

**Comment by queenofcorgis** [comment]
Thanks for sharing this idea @sospedra - I've labeled this so it can be reviewed by the proper teams!

**Comment by Khushboqof** [comment]
Yes, it's good, but it's a mistake in private projects between companies
